ts/api/api.ts

```diff
@@ -120,12 +121,17 @@ export interface OpenYoloApi {
    *    A promise for the response data from the authentication system.
    */
   proxyLogin(credential: Credential): Promise<ProxyLoginResponse>;
+
+  /**
+   * Cancels a pending request.
+   */
+  cancel(): void;
```

Comment:
cancelLastRequest() would be more descriptive, and I'm not sure if it should perhaps return a Promise so that developers can await the cancellation before triggering a new request.